scripts/mmvec

```diff
@@ -96,14 +96,18 @@ def mmvec():
               help=('Path to save the ordination learned from the model. '
                     'If this is not specified, then this will be saved under '
                     '`--summary-dir`.'))
+@click.option("--equalize-biplot", default=False, required=False,
+              help=('Equalize the norms of the singular'
```

Comment:
```suggestion
              help=('Equalize the norms of the singular '
```
This needs spacing, otherwise it will look like `singularvectors` on the command line interface.